docs/integrations/flutter-specific-workflow-steps/flutter-analyze.md

```diff
@@ -0,0 +1,39 @@
+---
+title: Flutter Analyze
+metaTitle: Flutter Analyze
+metaDescription: Flutter Analyze
+sidebar_position: 3
+---
+
+import Screenshot from '@site/src/components/Screenshot';
+
+# Flutter Analyze
+
+This component runs the `flutter analyze` command in your Flutter project. Please note that it requires the preceding [**Flutter Install**](https://docs.appcircle.io/workflows/flutter-specific-workflow-steps#flutter-install) and [**Git Clone**](https://docs.appcircle.io/workflows/common-workflow-steps/#git-clone) steps to run.
+
+### Prerequisites
+| Prerequisite Workflow Step                      | Description                                     |
+|-------------------------------------------------|-------------------------------------------------|
+| [**Git Clone**](https://docs.appcircle.io/workflows/common-workflow-steps/#git-clone) | This step will clone your project through the connected git provider and create the `$AC_REPOSITORY_DIR` variable. |
+| [**Flutter Install**](https://docs.appcircle.io/workflows/flutter-specific-workflow-steps#flutter-install) | This step will install the Flutter SDK. If a version is not specified, it will install the latest **stable** version. |
```

Comment:
This information can also be added:
```suggestion
| [**Flutter Install**](https://docs.appcircle.io/workflows/flutter-specific-workflow-steps#flutter-install) | This step will install the [Flutter SDK version](https://flutter-ko.dev/development/tools/sdk/releases). If a version is not specified, it will install the latest **stable** version. |
```